docs/src/rules/class-methods-use-this.md

```diff
@@ -408,3 +444,9 @@ class Derived implements Base {
 ```
 
 :::
+
+
+## When Not To Use It
+
+Fixing violations of this rule almost always is a breaking change, since it requires a change at every usage of the affected method.
+Therefore, if your project has downstream consumers you cannot break, or you simply do not wish to make invasive changes to every call site of a method, it likely does not make sense to address violations of this rule.
```

Comment:
[Approachability] Similarly here, it's not often a "simple" choice. That word doesn't add any meaning either:

```suggestion
Therefore, if your project has downstream consumers you cannot break, or you do not wish to make invasive changes to every call site of a method, it likely does not make sense to address violations of this rule.
```